.travis.yml

```diff
@@ -1,27 +1,23 @@
 dist: xenial
 sudo: required
 
-language: minimal
+language: node_js
 
-env:
-  # At the moment, only Node.js v8 is enabled, because of stability issues with
-  # Travis CI + Xenial causing builds to randomly fail. Bigger matrices mean
-  # higher chances of having some jobs failing.
-  # - NODE_VERSION=9 # Current stable
-  - NODE_VERSION=8 # Active LTS until April 2019
-  # - NODE_VERSION=6 # Active LTS until April 2018
+node_js:
+- 8 # Active LTS until April 2019
```

Comment:
This PR specifically targets one Node version, as said version gets packaged to the final package.

If we specify v8, we verify that Lounge runs correctly on that version, and it will be used in the final .deb.

Reply:
That makes complete sense, thanks!